npm_and_yarn/spec/dependabot/npm_and_yarn/file_parser_spec.rb

```diff
@@ -59,6 +59,12 @@
         its(:length) { is_expected.to eq(0) }
       end
 
+      context "with no lockfile, and yarn `workspace:` requirements" do
```

Comment:
💅 I first read this to mean "with no lockfile and no yarn `workspace:` requirements". I think this re-ordering makes the intention of the spec 🤏 clearer. What do you think?

```suggestion
      context "with yarn `workspace:` requirements and no lockfile" do
```